tests/storage-metadata

```diff
@@ -234,12 +237,17 @@ for vers in "latest" "5.21" "5.0"; do
         waitInstanceReady i1
         lxc stop -f i1
 
-        # Only try resetting the config keys if the local version of LXD (${LXD_SNAP_CHANNEL}) isn't 5.0.
-        if [ "${vers}" = "5.0" ] && [ "$(echo "${LXD_SNAP_CHANNEL}" | cut -d/ -f1)" != "5.0" ]; then
-            echo "==> Reset config keys not available in 5.0 to enable migration"
+        # Unset incompatible volatile key if local version of LXD doesn't support instance_ready_state.
+        if ! lxc exec c1 -- sh -c "$(declare -f hasNeededAPIExtension); hasNeededAPIExtension instance_ready_state"; then
             lxc config unset i1 volatile.last_state.ready
         fi
 
+        # Unset incompatible volatile key if local version of LXD doesn't support vm_persistent_bus.
+        if ! lxc exec c1 -- sh -c "$(declare -f hasNeededAPIExtension); hasNeededAPIExtension vm_persistent_bus"; then
+            lxc config unset i1 volatile.bus.mode
+            lxc config unset i1 volatile.eth0.bus
```

Comment:
Worth using `lxc config set i1 volatile.bus.mode="" volatile.eth0.bus=""` to bulk the changes?